static/sass/styles.scss

```diff
@@ -242,3 +242,9 @@ abbr[title] {
   height: auto !important;
   width: 100% !important;
 }
+
+// Override for wordpress use of <figure>
+.blog-article figure {
+  max-width: 37rem;
```

Comment:
Can you use `map-get($max-widths, default);` so it stays inline with Vanilla updates.

Reply:
that map doesn't exist on ubuntu.com yet

Reply:
@pmahnke You're right, the map is introduced in v2. For future-proofing, I'd use `%max-width--p` and it'll get caught when upgrading to v2